docs/uptime/monitoring-and-maintaining-your-server-new-manager/index.md

```diff
@@ -0,0 +1,185 @@
+
+### Assess Your Needs
+
+Not everyone needs to monitor the availability of their server. For example, if you use your Linode to host a personal picture gallery website for friends and family, the occasional service interruption probably won't bother you. The small inconvenience of your website going offline for a few minutes doesn't justify the time it would take to set up and configure an availability monitoring tool.
+
+If you depend on your website or web application for your livelihood, an availability monitoring tool is practically a necessity. Once set up, the tool actively watches your servers and services and alerts you when they're unavailable. You'll be able to troubleshoot the problem and restore service as quickly as possible.
+
+Whether you use one Linode or dozens of them, mission-critical servers and services should be watched by an independent monitoring tool that can keep tabs on their availability. The tool should have an automated method of detecting service-related incidents and be able to notify you via email, text message, or SMS. That way you'll know that a server or service is down within minutes of it having failed.
+
+### Find the Right Tool
+
+There are several different availability monitoring tools available. Your decision should be based on how many servers you'll be monitoring:
+
+-   **Multiple Servers**: If you run more than one server, the [Elastic Stack](/docs/databases/elasticsearch/visualize-apache-web-server-logs-using-elastic-stack-on-debian-8/) is an excellent monitoring tool.
+-   **Single Server**: If you only run a single server, you might want to use a third-party service to monitor your Linode. You could also use a network diagnostic tool like [MTR](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) to diagnose and isolate networking errors.
+-   **Linode Longview**: [Longview](/docs/platform/longview/longview/) is Linode's own monitoring and graphing service. It offers real time data that can be used to help identify system issues. Using Longview in conjunction with a notification service can help you to identify issues faster, and get your system up and running sooner.
+
+### Configure Shutdown Watchdog
+
+Shutdown Watchdog, also known as *Lassie*, is a Linode Manager feature capable of automatically rebooting your Linode if it powers off unexpectedly. Lassie is not technically an availability monitoring tool, but it can help get your Linode back online fast if it's accidentally powered off.
+
+To turn Lassie on and off:
+
+1.  Log in to the [Linode Manager](https://manager.linode.com).
```

Comment:
cloud.linode.com. This could be a constant so that you only have to change it once when the final switchover happens